apps/camera/js/controllers/camera.js

```diff
@@ -90,6 +91,18 @@ CameraController.prototype.bindEvents = function() {
   debug('events bound');
 };
 
+CameraController.prototype.onRecordingChange = function(recording) {
+  var active;
+  if (recording === 'started') {
```

Comment:
You can probably shorten this up a bit. Not super important, but it might be a little easier to read:
```
CameraController.prototype.onRecordingChange = function(recording) {
  if (recording === 'started') {
    this.app.set('recording', true);
  } else if (recording === 'stopped') {
    this.app.set('recording', false);
  }
};
```